plugins/web/opentelemetry-instrumentation-user-interaction/test/userInteraction.test.ts

```diff
@@ -40,6 +40,12 @@ const FILE_URL =
   'https://raw.githubusercontent.com/open-telemetry/opentelemetry-js/main/package.json';
 
 describe('UserInteractionInstrumentation', () => {
+  afterEach(() => {
+    while (document.body.lastChild) {
+      document.body.removeChild(document.body.lastChild);
```

Comment:
I'm not sure why this is necessary?  Perhaps a comment?

Reply:
Comment added.